package/system-role-hpc-server.changes

```diff
@@ -4,7 +4,7 @@ Fri Apr 20 08:57:52 UTC 2018 - cgoll@suse.com
 - changed required pattern from minimal_base to base bsc#1090282 
 - added subvolume description to installation.xml to make btrfs
   available on guided partitioner (bsc#1090380)
-- changed workload_server to hpc_workload_server (bsc#108951)
+- changed workload_server to hpc_workload_server (bsc#1089511)
 
```

Comment:
Please, add the version number here, as we do in other role packages.

Reply:
Sorry, I forgot to paste [this link](https://github.com/yast/system-role-server-default/blob/master/package/system-role-server-default.changes#L5).